### DISCUSSION TITLE
Manage Version and Build Number

### DISCUSSION THREAD

**Discussion by marksmanalo** [fastlane/fastlane#19413] [Q&A]
In iOS when archiving and uploading to App Store Connect, there is an option to 'Manage Version and Build Number'
Does the fastlane action 'upload_to_testflight' support unchecking/checking this option?

![image](https://user-images.githubusercontent.com/39017378/135544891-ba4c6718-e1aa-428e-ba48-01136bbcf234.png)

**Comment by mobielbekeken-thijs** [comment]
You can fix this by setting 'manageAppVersionAndBuildNumber ' to false in the export_options parameter when building the app, e.g.:

```
build_app(
  export_options: {
    manageAppVersionAndBuildNumber: false
  }
)
```
